packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Apply/Devfile/index.tsx

```diff
@@ -206,7 +249,38 @@ class StepApplyDevfile extends AbstractLoaderStep<Props, State> {
     );
   }
 
+  private handleCreateWorkspaceError(): void {
+    const { defaultDevfile } = this.props;
+    const { devfile } = this.state;
+    const _devfile = defaultDevfile;
```

Comment:
You need to clone the default devfile, otherwise, it changes, isn't it?

```suggestion
    const _devfile = cloneDeep(defaultDevfile);
```

Reply:
It doesn't change because of this [line](https://github.com/eclipse-che/che-dashboard/pull/643/files#diff-a5725d88e5b0467a23748655268db89c00a2f5e7915a517a344f66a9b657bcb1R89):
```
return load(devfileContent) as devfileApi.Devfile;
```

Reply:
You're right. Thanks